core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/AuthorizationManager.java

```diff
@@ -197,6 +197,19 @@ void clearRoleActionOnAllResources(String roleName, String action)
     String[] getAllowedUIResourcesForUser(String userName, String permissionRootPath)
             throws UserStoreException;
 
+    /**
+     * Returns the complete set of resources allowed for Role.
+     *
+     * @param roleName
+     * @return
+     * @throws UserStoreException
+     */
+    default String[] getAllowedUIResourcesForRole(String roleName, String permissionRootPath)
```

Comment:
This is not needed since it is inherited from org.wso2.carbon.user.api.AuthorizationManager